resources/views/components/remove-member-modal.blade.php

```diff
@@ -0,0 +1,30 @@
+<div x-data="modal()">
+    <button class="secondary" type="button" @click="showModal">{{ __('Remove') }}</button>
+    <template x-teleport="body">
+        <div class="modal-wrapper" x-show="showingModal">
+            <div class="modal stack" @keydown.escape.window="hideModal">
+                <h3>{{ __('Remove :member from your organization', ['member' => $member->name]) }}</h3>
+
+                <p>
+                    {{ __('Are you sure you want to remove :member from :organization? You cannot undo this.', ['member' => $member->name, 'organization' => $membershipable->name]) }}
```

Comment:
If the member is the current user, it this should say "Are you sure you want to leave :organization?".